modules/userdirectory/src/main/java/org/opencastproject/userdirectory/UserAndRoleDirectoryServiceImpl.java

```diff
@@ -336,17 +352,30 @@ private User loadUser(Tuple<String, String> orgUser) {
     for (Role role : roles) {
       if (Role.Type.EXTERNAL_GROUP.equals(role.getType())) {
         // Load roles granted to this group
-        logger.debug("Resolving transitive roles for user {} from external group {}", user.getUsername(), role.getName());
+        logger.debug(
+            "Resolving transitive roles for user {} from external group {}",
+            user.getUsername(),
+            role.getName()
+        );
         for (RoleProvider roleProvider : roleProviders) {
           if (roleProvider instanceof GroupProvider) {
             List<Role> groupRoles = ((GroupProvider) roleProvider).getRolesForGroup(role.getName());
             if (groupRoles != null) {
               for (Role groupRole : groupRoles) {
                 derivedRoles.add(JaxbRole.fromRole(groupRole));
               }
-              logger.debug("Adding {} derived role(s) for user {} from internal group {}", derivedRoles.size(), user.getUsername(), role.getName());
+              logger.debug(
+                  "Adding {} derived role(s) for user {} from internal group {}",
+                  derivedRoles.size(),
```

Comment:
I would join these as well.  In general, the form
```
debug(
"my logging statement with random includes {} {} {}",
var1, var2, var3);
```
parses best to my eyes, although I get that autoformatting that is probably hard :D